packages/shared/src/lib/featureManagement.ts

```diff
@@ -122,8 +128,30 @@ export class Features {
 
   static readonly MyFeedOnboardingVersion = new Features(
     'my_feed_onboarding_version',
-    'control',
-    ['control', 'v1', 'v2', 'v3'],
+    MyFeedOnboardingVersionEnum.Control,
```

Comment:
We recently decided to move away from control as a name.
Can you please move them all one up, but perhaps let product know about it so they know which version is which naming.

Reply:
The idea behind this change is that `control` is a generic product term. Today it might reference v1, but in 3 months from now they might decide v2 becomes the control version. (Hope this makes sense)